frontend/src/components/messages/MessagesTab.vue

```diff
@@ -5,15 +5,15 @@
         <v-col cols="12">
           <AppButton size="small" class="messages-button" @click="toggleNewRequestDialog()">
             <v-icon left>mdi-email-plus-outline</v-icon>
-            New message
+            <span>New message</span>
```

Comment:
Hi Jen, could we remove the span tag in Notification's buttons as well so they will look consistent?

Reply:
done :)